rest_api/operator_apis/kubeapiserver-operator-openshift-io-v1.adoc

```diff
@@ -98,7 +98,7 @@ Type::
 
 | `unsupportedConfigOverrides`
 | ``
-| unsupportedConfigOverrides holds a sparse config that will override any previously set options.  It only needs to be the fields to override it will end up overlaying in the following order: 1. hardcoded defaults 2. observedConfig 3. unsupportedConfigOverrides
+| The `unsupportedConfigOverrides` object holds a minimal configuration that replaces any previously set option with the newer option unless you explicitly specify both options. By order of precedence, options specified for the `KubeAPIServer` that are either hardcoded or set by the `observedConfig` object override options set by the `unsupportedConfigOverrides` object.
```

Comment:
Using override is accurate, not `replaces any previously`, in original description, `it will end up overlaying in the following order: 1. hardcoded defaults 2. observedConfig 3. unsupportedConfigOverrides`, my understanding is the `hardcoded defaults` config will be overrided by `observedConfig`, then `observedConfig` will be overrided by `unsupportedConfigOverrides`

Reply:
/assign @sanchezl 
What do you think of this?

Reply:
Thanks, @wangke19 . I updated the content based on your reply. I'm discussing tje update further with Luis.

Reply:
From a customer/user perspective the documentation already provides too much detail, given that use of this field is unsupported. The original description is correct ([here is the code that does the merging](https://github.com/openshift/cluster-kube-apiserver-operator/blob/a9a4df5f0cedeba2868a6ceb63ab58f67a0cdbf2/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L220-L223)). That said, there, are other uses of the `unsupportedConfigOverrides` in the kubeapiserver-operator that simply examine this field for options.

If I had "clarify" this description, it would be as follows:

> `unsupportedConfigOverrides` can be used to add configuration changes that are not supported by a cluster operator, typically at the direction of customer support for testing or troubleshooting purposes. Using this property could potentially cause unexpected behavior or conflict with other configuration options. Using this property will block cluster upgrades. You must remove this setting before you can upgrade your cluster.

Reply:
Thanks, @sanchezl .

Yes, the name of the property says it all! I agree that the original description provides too much detail and does not cover all possible use cases. 

I replaced the original text with your suggested text (albeit with some minor changes). Please let me know if you approve of the changes, and I'll send this PR to QE for review.